e2e/services/navigation.service.ts

```diff
@@ -0,0 +1,63 @@
+import { browser, ExpectedConditions as EC } from 'protractor';
+import { LoginPage } from '../page-objects/login/login.po';
+import { MainPage } from '../page-objects/main/main.po';
+import { BasePage } from '../page-objects/base-page';
+import { TestUtil } from '../utilities/test-util';
+import { ButtonState, FormService } from './form.service';
+
+declare const allure: any;
+
+export class NavigationService {
```

Comment:
We do have a navigation service for each functional area. Otherwise, when there's so many screens, this file will be too long. According to that, I recommend to create 3 navigation services for the login, patient and allergy.

Reply:
My idea is to have widgets and PO. And for each PO create a different services.